docs/streamtypes/tile-document/api.md

```diff
@@ -0,0 +1,182 @@
+| `deterministic` | optional    | boolean          | If false, allows TileDocuments with the same `content` and `metadata` to generate unique StreamIDs | If empty, defaults to false |
+
+[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_stream_tile.tilemetadataargs-1.html){:target="_blank"}
+
+
+!!! info "Using the `deterministic` parameter"
+    For most use cases you will likely want to leave the `deterministic` parameter set to false. However for special circumstances, you may want this to be set to true. For example this should be set to true if you would like to enable [deterministic queries](#query-a-deterministic-document) for your document. If this is your use case, then it is also important that you think very carefully about what `content` is provided when first creating the document, as that content is what will need to be provided again in order to deterministically query the document. It is recommended that the document be created with the minimal possible content to uniquely identify the document. You can proceed to add additional content to your document by [updating it](#update-a-document).
+
+[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_stream_tile.tilemetadataargs-1.html#deterministic){:target="_blank"}
+
+
+#### opts (optional)
+The final argument to `TileDocument.create` is an instance of [`CreateOpts`](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.createopts-1.html){:target="_blank"}, which are options that control network behaviors performed as part of the operation.  They are not included in the document itself.
+
+| Parameter     | Required?   | Value            | Description | Default value |
+| ------------- | ----------- | ---------------- | ----------- | ----- |
+| `anchor`      | optional    | boolean          | Request an anchor after creating the document | true |
+| `publish`     | optional    | boolean          | Publish the new document to the network | true |
+| `sync`        | optional    | enum             | Controls behavior related to syncing the current document state from the network | PREFER_CACHE |
+| `syncTimeoutSeconds` | optional    | number            | How long to wait to hear about the current state of the document from the network | 3 for deterministic documents, 0 otherwise |
+
+[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.createopts-1.html){:target="_blank"}
+
+
+## Update a document
+Use the [`doc.update()`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_tile.tiledocument-1.html#update){:target="_blank"} method to update the `content` or `metadata` of an existing TileDocument.
+
+```javascript
+const doc = await TileDocument.create(ceramic, content, metadata, opts)
```

Comment:
I would do this instead to make it clear that you can updating an existing document:
```suggestion
const doc = await ceramic.loadStream<TileDocument>(docid)
```